src/tests/ftest/recovery/container_cleanup.py

```diff
@@ -34,9 +34,9 @@ def test_container_label_inconsistency(self):
 
         Jira ID: DAOS-12289
 
-        :avocado: tags=all,pr
-        :avocado: tags=vm
-        :avocado: tags=recovery,container_cleanup
+        :avocado: tags=all,daily_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=recovery,test_cat_recovery,container_cleanup
         :avocado: tags=ContainerCleanupTest,test_container_label_inconsistency
```

Comment:
Intentionally moved from VM to HW?

Reply:
Yes, CR tests are relative slow, run in VM will be more slow, let's test in HW.

Reply:
This is a tricky decision. HW is faster than VMs, but VMs are cheaper and we don't have as much contention for VM resources. HW Medium has the most contention and takes the most time overall. So unless the tests are VMs are significantly slower than on HW, we should keep them on VMs

Reply:
We have many cat_recovery test cases under the recovery directory, most of with tag "hw". It is unnecessary to split them into "VM" and "HW". I prefer to unify them.

Reply:
> It is unnecessary to split them into "VM" and "HW"

I disagree. If a test can run on VMs, we should run it on VMs to save HW resources. Moving tests to HW contributes to CI slowness

Reply:
> some time-consuming test cases run on "VM" may need 2 hours, but run on "HW" may just take 30 minutes

Yes, I agree that could be a reason to move to HW. And we have done similar in the past.
Looking at a master run, where these tests are on VM:
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/master/9652/testReport/FTEST_recovery/

And yours, where they are on HW:
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-14161/3/testReport/FTEST_recovery/

They are indeed faster on HW.

Not a blocker, but ideally we should reduce the test timeouts if they are running faster now.